### ISSUE TITLE
Combobox: Fix the docs JS to match true A11Y for DS4

### ISSUE THREAD

**Issue by seangates** [eBay/skin#345]
Currently the DS4 version of the combobox in the documentation doesn't work, and much of the A11Y functionality is broken.

**Comment by ianmcburnie** [comment]
I've decided to strip out the JS functionality (leaving only basic expand/collapse for now). This is an interim step while I clear out all jquery code and unify the DS4 and DS6 sites on vanilla js only. I will add links to MIND patterns for detailed example of behaviour.

Closing ticket.
